capybara-webkit.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = ">= 1.9.0"
 
-  s.add_runtime_dependency("capybara", ">= 2.3.0", "< 2.5.0")
+  s.add_runtime_dependency("capybara", ">= 2.3.0")
```

Comment:
I'm not sure that is right. 3.0 may break but not definitely will. If we introduce max gem version  — we lock feature usage without reason.